src/hotspot/share/utilities/elfFile.hpp

```diff
@@ -25,7 +25,7 @@
 #ifndef SHARE_UTILITIES_ELFFILE_HPP
 #define SHARE_UTILITIES_ELFFILE_HPP
 
-#if !defined(_WINDOWS) && !defined(__APPLE__) && !defined(_AIX)
+#if !defined(_WINDOWS) && (!defined(__APPLE__) || defined(__BIOS__)) && !defined(_AIX)
```

Comment:
Is this really correct?

Reply:
No, that's a typo indeed. 
With `make static-libs`, this code is not yet included but it is needed in the next step (when we use `make static-libs-image` which will also build the VM.
I fixed this.

Reply:
That explains why it was not discovered. Thanks, I was just wondering that.